package/pyproject.toml

```
@@ -4,13 +4,10 @@ requires = [
     "Cython>=0.28",
     "packaging",
     # lowest NumPy we can use for a given Python,
-    # except for more exotic platforms (linux/Mac ARM flavors)
-    "numpy==1.19; python_version=='3.8' and (platform_machine!='arm64' or platform_system!='Darwin') and platform_machine!='aarch64'",
-    "numpy==1.19.3; python_version=='3.9' and (platform_machine!='arm64' or platform_system!='Darwin') and platform_machine!='aarch64'",
+    # arm65 on Darwin supports py3.8 and above and requires numpy>=1.20.0
```

Is "arm65" a thing? OR should this be "arm64"?

---

typo definitely

---

```suggestion
    # arm64 on Darwin supports py3.8 and above and requires numpy>=1.21.0
```
and then change below